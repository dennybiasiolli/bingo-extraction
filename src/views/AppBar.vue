<template>
  <v-app-bar app color="primary" dark>
    <div class="d-flex align-center">
      <v-img
        alt="Vuetify Logo"
        class="shrink mr-2"
        contain
        src="https://cdn.vuetifyjs.com/images/logos/vuetify-logo-dark.png"
        transition="scale-transition"
        width="40"
      />

      <h1>Bingo Extraction</h1>
    </div>

    <v-spacer></v-spacer>

    <h4>Version 0.0.1</h4>
  </v-app-bar>
</template>
