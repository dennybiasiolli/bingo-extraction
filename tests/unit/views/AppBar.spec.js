import { shallowMount } from '@vue/test-utils';
import AppBar from '@/views/AppBar.vue';

describe('AppBar.vue', () => {
  it('renders as expected', () => {
    const wrapper = shallowMount(AppBar, {
      stubs: ['v-app-bar', 'v-img', 'v-spacer'],
    });
    expect(wrapper).toMatchSnapshot();
  });
});
