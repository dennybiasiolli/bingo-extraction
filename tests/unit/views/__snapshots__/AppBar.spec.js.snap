// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppBar.vue renders as expected 1`] = `
<v-app-bar-stub app="" color="primary" dark="">
  <div class="d-flex align-center">
    <v-img-stub alt="Vuetify Logo" contain="" src="https://cdn.vuetifyjs.com/images/logos/vuetify-logo-dark.png" transition="scale-transition" width="40" class="shrink mr-2"></v-img-stub>
    <h1>Bingo Extraction</h1>
  </div>
  <v-spacer-stub></v-spacer-stub>
  <h4>Version 0.0.1</h4>
</v-app-bar-stub>
`;
